src/bios/flib/src/video_io/io.rs

```diff
@@ -15,14 +15,32 @@ pub fn cprint_info(str: &[u8]) {
     }
 }
 
+pub fn write(x: u16, y: u16, color: u8) {
+    unsafe {
+        asm!(
+        "pusha",
+        "mov ah, 0x0c",
+        "mov bh, 0x00",
+        "int 0x10",
+        "popa",
+        in("dx") y,
+        in("cx") x,
+        in("al") color
+        )
+    }
```

Comment:
```suggestion
pub fn write(x: u16, y: u16, color: u8) {
    let ax: u16 = u16::from(color) | 0x0c00
    unsafe {
        asm!(
        "push ebx",
        "mov ah, 0x0c",
        "mov bh, 0x00",
        "int 0x10",
        "pop ebx",
        in("dx") y,
        in("cx") x,
        in("ax") ax
        )
    }
```
ça devrait suffire de juste push `ebx`